google/auth/external_account_authorized_user.py

```diff
@@ -73,6 +73,7 @@ def __init__(
         token_url=None,
         token_info_url=None,
         revoke_url=None,
+        scopes=None,
```

Comment:
token_url: "... for refresh" in arg documentation below

Reply:
Done